.babelrc

```diff
@@ -1,6 +1,7 @@
 {
   "presets": [
     "env",
-    "react"
+    "react",
+    "stage-2"
```

Comment:
Should swap out for stage-3 plugin since its newer

Reply:
Did you mean stage-0?  It looks like it includes features from 1, 2 and 3.

http://babeljs.io/docs/plugins/preset-stage-0

Reply:
Oo yeah, good catch. I always mix up the order they follow. Stage-3 has all the candidate features